ext/mysqli/mysqli_fe.c

```diff
@@ -261,6 +261,7 @@ ZEND_END_ARG_INFO()
 ZEND_BEGIN_ARG_INFO_EX(arginfo_mysqli_query, 0, 0, 2)
 	MYSQLI_ZEND_ARG_OBJ_INFO_LINK()
 	ZEND_ARG_INFO(0, query)
+	ZEND_ARG_INFO(0, resultmode)
```

Comment:
@motecshine `resultmode` is actual name of this parameter and should not be changed.
check this one for details https://github.com/php/php-src/blob/c8aa6f3a9a3d2c114d0c5e0c9fdd0a465dbb54a5/ext/mysqli/mysqli_nonapi.c#L550